pkg/stanza/operator/helper/multiline.go

```diff
@@ -96,13 +96,17 @@ func (c MultilineConfig) getSplitFunc(enc encoding.Encoding, flushAtEOF bool, fo
 
 // NewLineStartSplitFunc creates a bufio.SplitFunc that splits an incoming stream into
 // tokens that start with a match to the regex pattern provided
-func NewLineStartSplitFunc(re *regexp.Regexp, flushAtEOF bool) bufio.SplitFunc {
+func NewLineStartSplitFunc(re *regexp.Regexp, flushAtEOF bool, preserveWhitespace bool) bufio.SplitFunc {
```

Comment:
nit: Might as well condense here too.
```suggestion
func NewLineStartSplitFunc(re *regexp.Regexp, flushAtEOF, preserveWhitespace bool) bufio.SplitFunc {
```